samples/go/nomdex/nomdex_test.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package main
+
+import (
+    "fmt"
+    "testing"
+
+    "github.com/attic-labs/noms/go/chunks"
+    "github.com/attic-labs/noms/go/datas"
+    "github.com/attic-labs/noms/go/marshal"
+    "github.com/attic-labs/noms/go/spec"
+    "github.com/attic-labs/noms/go/types"
+    "github.com/attic-labs/noms/go/util/clienttest"
+    "github.com/attic-labs/testify/suite"
+)
+
+func TestCSVImporter(t *testing.T) {
+    suite.Run(t, &testSuite{})
+}
+
+type TestObj struct {
+    Key  int
+    Fname string
+    Lname string
+    Gender string
+    Age int
+}
+
+type testSuite struct {
+    clienttest.ClientTestSuite
+}
+
+func TestNomdex(t *testing.T) {
+    suite.Run(t, &testSuite{})
+}
+
+func makeTestDb(s *testSuite, dsId string) datas.Database {
+    db := datas.NewDatabase(chunks.NewLevelDBStore(s.LdbDir, "", 1, false))
+    fmt.Println("ldbDir:", s.LdbDir)
+    l1 := []TestObj{
+        TestObj{1, "will", "smith", "m", 40},
```

Comment:
@mikegray just fixed a bunch of these. The struct name should be omitted here.
go report finds these... I'm not sure how to run that locally.